### PR TITLE
Remove mention of `reverse` plugin

### PR DESCRIPTION
**Pull request by stp-ip**
Use template plugin within fallthrough usage example

Fixes: coredns/coredns/issues/2978

### CODE REVIEW

File: content/manual/plugins-dev.md [hunk 1]
```diff
@@ -119,11 +119,11 @@ a query comes in that is not a subdomain (or equal to) `example.org` the next pl
 
 Now, the world isn't perfect, and there are good reasons to "fallthrough" to the next middleware,
 meaning a plugin is only responsible for a *subset* of names within the zone. The first of these
-to appear was the *reverse* plugin that synthesizes PTR and A/AAAA responses (useful with IPv6).
+to appear was the *template* plugin that synthesizes various responses.
```
Comment (johnbelamaric) [inline content/manual/plugins-dev.md]: Well...perhaps pedantic but this revises history. What about:

...was the *reverse* plugin, now replaced with the *template* plugin that synthesizes various responses.

File: content/manual/plugins-dev.md [hunk 2]
```diff
@@ -119,11 +119,11 @@ a query comes in that is not a subdomain (or equal to) `example.org` the next pl
...
 
-The nature of the *reverse* plugin is such that it only deals with A/AAAA and PTR, and then only
-for a subset of the names. Ideally, you would want to layer *reverse* **in front of** another
-plugin such as *file* or *auto* (or even *proxy*). This means *reverse* handles some special
+The nature of the *fallthrough* plugin is such that it only deals with a specified record TYPE, and then only
```
Comment (johnbelamaric) [inline content/manual/plugins-dev.md]: that's not true

Comment (stp-ip) [reply content/manual/plugins-dev.md]: Good catch. Wrong thought written down. Should be fixed.

File: content/manual/plugins-dev.md (hunk 1 above)
Comment (stp-ip) [reply content/manual/plugins-dev.md]: Thanks. Fixed.

**Comment by miekg** [comment]
We should setup OWNERS here (that how the bot is triggered)

**Comment by miekg** [comment]
git pull + make on serving host

**Pull request merged**